### DISCUSSION TITLE
Mac Context Menu

### DISCUSSION THREAD

**Discussion by johnstonmatt** [johnlindquist/kit#181] [Ideas]
I built a script in shell as a MacOS service, can Kit enable the distribution of MacOS services?
Services can be launched from the context menu, one example is "Right click to open in VSCode"
Here is a link to the repo for that service:

https://github.com/johnstonmatt/open-in-vscode

It would benefit from the distribution and discoverability of the Kit registry :D

**Comment by johnlindquist** [comment]
Sure, I made a video/post about using Automator to wrap a Script Kit script:

https://github.com/johnlindquist/kit/discussions/143

I'll remove the automator step in the future so you can do it directly from Script Kit sometime in the future.
